lib/taste_tester/commands.rb

```diff
@@ -210,5 +210,179 @@ def self.upload
       logger.error(exception.backtrace.join("\n"))
       exit 1
     end
+
+    def self.impact
+      # Get the repository based on the paths specified in config.rb
+      repo = BetweenMeals::Repo.get(
+        TasteTester::Config.repo_type,
+        TasteTester::Config.repo,
+        logger,
+      )
+      if repo && !repo.exists?
+        fail "Could not open repo from #{TasteTester::Config.repo}"
+      end
+
+      # Compute the set of changes using Between Meals
+      changeset = _find_changeset(repo)
+
+      # Use the changeset computed earlier to find modified roles
+      if TasteTester::Config.use_custom_impact_hook
```

Comment:
Instead why not just do:

```
TasteTester::Hooks.impact_post_process(_find_impact(changeset))
```

Reply:
Instead of what? Do you mean in place of the custom_impact hook?

Reply:
yeah. See the expanded version of this in the other comment.